extensions/adobe/experience/decisioning/proposition.schema.json

```diff
@@ -0,0 +1,96 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/proposition",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Proposition",
+  "type": "object",
+  "meta:abstract": true,
+  "meta:extensible": true,
+  "description": "The  Decision Proposition XDM is used to capture the outcome of a decisioning process",
+  "definitions": {
+    "proposition-detail-datatype": {
+      "properties": {
+        "xdm:activity": {
+          "title": "Decision Activity",
+          "description": "Snapshot of the activity for which the proposition was made. The object contains at least the @id property value which is the URI of the decision activity. See schema https://ns.adobe.com/experience/decisioning/activity",
+          "$ref": "https://ns.adobe.com/xdm/data/record"
+        },
+        "xdm:selections": {
+          "title": "Selected Options",
+          "description": "Snapshots of the options that were selected for this activity. Those options are ususally constrained. The objects contain a at least the @id property value which is the URI of an option. See schema https://ns.adobe.com/experience/decisioning/option",
+          "type": "array",
+          "items": {
+            "$ref": "https://ns.adobe.com/xdm/data/record"
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "xdm:fallback": {
+          "title": "Fallback Option",
+          "description": "Snapshot of the fixed fallback option that was selected because no available option satisfied all constaints. The object contains at least the @id property value which is the URI of the fallback action. See schema https://ns.adobe.com/experience/decisioning/fallback-option",
+          "$ref": "https://ns.adobe.com/xdm/data/record"
+        }
+      }
+    },
+    "proposition-metrics-datatype": {
```

Comment:
Break out all of these definitions into separate schemas.

Reply:
What is the naming convention to give to those files? I have a datatype definition, a mixin definition and another supporting datatype definition (for metrics).

Reply:
I have broken up the proposition.schema.json into two parts, the original for the mixin and a second one for the proposition-detail datatype